build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -75,6 +75,14 @@ Below is an example for ads.
 
 [/example][/filter]
 
+### Standalone use outside the AMP environment (optional)
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide [Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/)
+
+#### Bento Example
+
+Code snippet of component usage in Bento.
```

Comment:
(without the backslashes) -
```suggestion
[example preview="top-frame" playground="true" imports="amp-__component_name_hyphenated__"]

\```html
Code snippet of component usage in Bento.
\```

[/example]
```

Reply:
I don't think amp.dev is able to support inline executable code samples for Bento at this time. So for now I think it might be best to keep it to code snippets vs full example

Reply:
Can you please test this out? I'm optimistic that using top-frame and disabling playground works.

Reply:
<img width="1427" alt="Screen Shot 2021-01-25 at 12 36 40 PM" src="https://user-images.githubusercontent.com/23297417/105763300-7c9e6880-5f0a-11eb-9089-c4a9c4619964.png">
I tried it out in the guide and I'm getting a lot of errors. I'm opted into the experimental channel and my glitch example is still working, so not too sure whats going on. Here's the code:


[example preview="top-frame" playground="false" imports="amp-accordion"]
```html
<amp-accordion id="my-accordion" disable-session-states>
  <section>
    <h2>Section 1</h2>
    <p>Content in section 1.</p>
  </section>
  <section>
    <h2>Section 2</h2>
    <div>Content in section 2.</div>
  </section>
  <section>
    <h2>Section 3</h2>
    <img class="article-img" width="1024" height="682" src="https://raw.githubusercontent.com/ampproject/amp.dev/future/examples/static/samples/img/product2_1024x682.jpg">
  </section>
</amp-accordion>
<button id="toggle">
  Toggle Accordion
</button>
<script>
  const toggleAccordion = async () => {
    const accordion = document.querySelector('#my-accordion');
    const api = await accordion.getApi();
    api.toggle();
  }
  const toggleButton = document.querySelector('#toggle');
  toggleButton.addEventListener('click', toggleAccordion);
</script>
```
[/example]